lib/pbench/server/database/models/datasets.py

```diff
@@ -311,17 +312,44 @@ def __str__(self) -> str:
 
 def current_time() -> datetime.datetime:
     """
-    Return the current time.
-
-    NOTE: we use this intermediary rather than binding the Column default
-    directly to datetime.datetime.now because that load-time binding defeats
-    unit test mocking. Binding it instead to this method allows a mock to
-    take effect.
```

Comment:
When I didn't see any reference, I suspect I was doing my search with a trailing open parenthesis...because searching without it finds the two you mentioned with no problem.  🥴 

> I could have done that with a simple `datetime.datetime.now`, but not with the `datetime.datetime.now(datetime.timezone.utc)` I need now.

Would [`datetime.datetime.utcnow()`](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) work?

Reply:
Or...if you really wanted...you could use a `lambda`:
```
12345678901234567890123456789012345678901234567890123456789012345678901234567890
    uploaded = Column(
        TZDateTime,
        nullable=False,
        default=lambda: datetime.datetime.now(datetime.timezone.utc))
```
🙃

Reply:
No, `.utcnow()` returns the current naive time in UTC; I want aware time. And I don't see any value in blowing up the Column definition with a long lambda.

Reply:
Ah, sorry -- I was confused:  I thought this was the context where we wanted a naive time.